esmgrids/cli.py

```diff
@@ -0,0 +1,39 @@
+import os
+import argparse
+
+from esmgrids.util import md5sum
+from esmgrids.mom_grid import MomGrid
+from esmgrids.cice_grid import CiceGrid
+
+
+def cice_from_mom():
```

Comment:
This looks good but I don't understand how to use it (from a test or the command line :))

Reply:
Sorry yes I found this :)